cmake/Modules/ObjectTargetHelpers.cmake

```diff
@@ -89,6 +136,30 @@ function(__target_link_objects_impl target linkage objlib is_inner_call)
       endif()
     endforeach()
   endif()
+  set(objlib_pilibs)
+  get_target_property(objlib_plibs "${objlib}" INTERFACE_LINK_LIBRARIES_PRIVATE)
+  if (objlib_pilibs)
```

Comment:
Should this be `objlib_plibs`? Just trying to match it up with the `objlib_ilibs`/`objlib_libs` section above.

Reply:
Yes, good catch. Fixed.